beamerthemeuulm.sty

```diff
@@ -198,131 +198,259 @@
 	\end{minipage}
 }
 
+% corresponds to '\halfpage', however, with the new columns we could use larger values
+% (same for thirdnext)
+\def\halfnext{.489\textwidth}
+% corresponds to '\thirdpage'
+\def\thirdnext{.31\textwidth}
+\def\uulm@cols@error{\PackageError{beamerthemeuulm}{The layout <TODO> you selected does not allow another \string\mynextcolumn\space (page: \thepage, frame: \insertframenumber,\on@line)}{Either remove the command, or choose another Layout}}
 
-\newcommand{\leftandright}[2]{
-    \halfpage{#1}
+\newcommand\leftandright[2]{%
+    \halfpage{#1}%
     \hfill
-    \halfpage{#2}
+    \halfpage{#2}%
 }
 
-\newcommand{\leftmiddleandright}[3]{
-	\thirdpage{#1}
+\newcommand\leftmiddleandright[3]{%
+    \thirdpage{#1}%
     \hfill
-    \thirdpage{#2}
+    \thirdpage{#2}%
     \hfill
-    \thirdpage{#3}
+    \thirdpage{#3}%
 }
 
-\newcommand{\leftthenright}[2]{
-    \halfpage{#1}
+\newcommand\leftthenright[2]{%
+    \halfpage{#1}%
     \hfill
-    \onslide<2->{\halfpage{#2}}
+    \onslide<2->{\halfpage{#2}}%
 }
 
-\newcommand{\rightthenleft}[2]{
+\newcommand\rightthenleft[2]{
     \onslide<2->{\halfpage{#1}}
     \hfill
     \halfpage{#2}
 }
 
-\newcommand{\leftmiddlethenright}[3]{
-	\thirdpage{#1}
+\newcommand\leftmiddlethenright[3]{%
+    \thirdpage{#1}%
     \hfill
-    \onslide<2->{\thirdpage{#2}}
+    \onslide<2->{\thirdpage{#2}}%
     \hfill
-    \onslide<3->{\thirdpage{#3}}
+    \onslide<3->{\thirdpage{#2}}%
 }
```

Comment:
Why do we want to ignore the third parameter here and duplicate content of the second?

Reply:
Fixed